docs/bokeh/source/docs/user_guide/basic/axes.rst

```diff
@@ -108,3 +108,15 @@ adding new glyph methods as well as when adding new axis objects with the
 
 .. bokeh-plot:: __REPO__/examples/basic/axes/twin_axes.py
     :source-position: above
+
+.. _ug_basic_axes_fixed:
+
+Fixed location axis
+-------------------
+
+By defatult Bokeh places axes on the sides of plots, but it is possible to
```

Comment:
```suggestion
By default Bokeh places axes on the sides of plots, but it is possible to
```

Reply:
oops, I missed seeing these before merging. Planning to add more examples, I will get to them in those PRs